test/system/252-quadlet.bats

```diff
@@ -283,4 +283,66 @@ EOF
     service_cleanup $QUADLET_SERVICE_NAME inactive
 }
 
+@test "quadlet - network" {
+    local quadlet_file=$PODMAN_TMPDIR/basic_$(random_string).network
+    cat > $quadlet_file <<EOF
+[Network]
+Label=foo=bar other="with space"
+EOF
+
+    run_quadlet "$quadlet_file"
+
+    service_setup $QUADLET_SERVICE_NAME
+
+    local network_name=systemd-$(basename $quadlet_file .network)
+    run_podman network ls
+    is "$output" ".*${network_name}.*bridge.*"
+
+    run_podman network inspect  --format "{{.Labels.foo}}" $network_name
+    is "$output" "bar"
+    run_podman network inspect  --format "{{.Labels.other}}" $network_name
+    is "$output" "with space"
+
+    service_cleanup $QUADLET_SERVICE_NAME inactive
+}
+
+# A quadlet container depends on a quadlet volume
+@test "quadlet - network dependency" {
+    local quadlet_network_file=$PODMAN_TMPDIR/dep_$(random_string).network
+    cat > $quadlet_network_file <<EOF
+[Volume]
+EOF
+
+    run_quadlet "$quadlet_network_file"
+
+    local network_service=$QUADLET_SERVICE_NAME
+    local network_name=systemd-$(basename $quadlet_vol_file .network)
+
+    local quadlet_file=$PODMAN_TMPDIR/user_$(random_string).container
+    cat > $quadlet_file <<EOF
+[Container]
+Image=$IMAGE
+Exec=top
+Network=$network_service
+EOF
+
+    # Volume should not exist
+    run_podman network ls
+    assert "$output" !~ ".*${network_name}.*"
```

Comment:
Let's use `network exists` for that check.

Reply:
Fixed